core/integration/storagetest/mutation_logs.go

```diff
@@ -54,8 +54,8 @@ func mustMarshal(t *testing.T, p proto.Message) []byte {
 	return b
 }
 
-// https://dev.mysql.com/doc/refman/8.0/en/datetime.html
-var minWatermark = time.Date(1000, 1, 1, 0, 0, 0, 0, time.UTC)
+// The lowest common denominator for the minimum timestamp supported between all implementations.
```

Comment:
It's not clear what the [LCD](https://en.wikipedia.org/wiki/Lowest_common_denominator) of min timestamps would be?

Reply:
LCD was the wrong word choice.
"Lowest watermark supported by all implementations" is more precise.